fhir-profile/src/main/java/com/ibm/fhir/profile/ProfileSupport.java

```diff
@@ -291,11 +294,8 @@ public static ElementDefinition getElementDefinition(String path) {
     }
 
     public static Map<String, ElementDefinition> getElementDefinitionMap(String url) {
-        Map<String, ElementDefinition> elementDefinitionMap = ELEMENT_DEFINITION_CACHE.get(url);
-        if (elementDefinitionMap == null) {
-            elementDefinitionMap = ELEMENT_DEFINITION_CACHE.computeIfAbsent(url, ProfileSupport::computeElementDefinitionMap);
-        }
-        return elementDefinitionMap;
+        Map<String, Map<String, ElementDefinition>> elementDefinitionMapCache = CacheManager.getCacheAsMap(ELEMENT_DEF_CACHE_NAME, ELEMENT_DEF_CACHE_CONFIG);
+        return elementDefinitionMapCache.computeIfAbsent(url, ProfileSupport::computeElementDefinitionMap);
```

Comment:
It'd be nice to get a test case for this using `default`.  I think we enable server registry by default. if not, we do so on tenant1.

Reply:
Sure.  The hardest part is creating a simple extension definition (surprisingly difficult).
I just tested in locally with an extension from https://alvearie.io/alvearie-fhir-ig/StructureDefinition-process-name.html ...maybe I'll just suck that one down locally (unless we have a similar one laying around somewhere already).

Reply:
just given we have a 'security' label on this, I think we should.

Reply:
ok, I pushed a new test class named TenantIsolationTest